src/main/java/org/repodriller/domain/CommitPerson.java

```diff
@@ -0,0 +1,95 @@
+package org.repodriller.domain;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+/**
+ * A POJO to track details about a person involved with a commit.
+ */
+public class CommitPerson {
+	public String name;
```

Comment:
I just access these fields directly. Do you want me to use getters instead or to make these fields final?